### PR TITLE
[#3167] Add table column sort indicator

### PR DESCRIPTION
**Pull request by Maurier**
Adds sort indicators to table columns where sorting is enabled.

**Before**
-----------------
image

**After**
-----------------
image

### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: theme/static/css/jquery.dataTables.min.css [hunk 1]
```diff
@@ -1 +1 @@
...
...l div.dataTables_scrollBody{*margin-top:-1px;-webkit-overflow-scrolling:touch}.dataTables_wrapper .dataTables_scroll div.dataTables_scrollBody th,.dataTables_wrapper .dataTables_scroll div.dataTables_scrollBody td{vertical-align:middle}.dataTables_wrapper .dataTables_scroll div.dataTables_scrollBody th>div.dataTables_sizing,.dataTables_wrapper .dataTables_scroll div.dataTables_scrollBody td>div.dataTables_sizing{height:0;overflow:hidden;margin:0 !important;padding:0 !important}.dataTables_wrapper.no-footer .dataTables_scrollBody{border-bottom:1px solid #111}.dataTables_wrapper.no-footer div.dataTables_scrollHead table,.dataTables_wrapper.no-footer div.dataTables_scrollBody table{border-bottom:none}.dataTables_wrapper:after{visibility:hidden;display:block;content:"";clear:both;height:0}@media screen and (max-width: 767px){.dataTables_wrapper .dataTables_info,.dataTables_wrapper .dataTables_paginate{float:none;text-align:center}.dataTables_wrapper .dataTables_paginate{margin-top:0.5em}}@media screen and (max-width: 640px){.dataTables_wrapper .dataTables_length,.dataTables_wrapper .dataTables_filter{float:none;text-align:center}.dataTables_wrapper .dataTables_filter{margin-top:0.5em}}
```
Comment (Maurier) [inline theme/static/css/jquery.dataTables.min.css]: For this file I just updated the relative path. Example: "../img/sort_both.png"

**Comment by Lizabrazil** [comment]
@cuahsimarko review

File: theme/static/css/jquery.dataTables.min.css (hunk 1 above)
Comment (cuahsimarko) [reply theme/static/css/jquery.dataTables.min.css]: this was changed in the minified file... is there a source for this or is this standard for changes in this area on HydroShare? My understanding is that source files undergo a build of sorts, in doing so get minified. Ok if we've just been doing it this way for a while and we can fix process another week/month.

Comment (cuahsimarko) [reply theme/static/css/jquery.dataTables.min.css]: @Maurier I left a comment about where this change lives and also have some questions about getting it to work locally and any nuances you may know about across NGINX, hsctl, collectstatic....

Comment (Maurier) [reply theme/static/css/jquery.dataTables.min.css]: We are hosting this script ourselves, so we are good to make changes to the relative path as needed. Similarly to our custom CSS, this script gets placed in the static folder in the deployment. The relative path should not change for deployed instances of the site so you should be good to use it locally. Let me know if there are any issues.

**Comment by cuahsimarko** [comment]
@Maurier please do a local check against origin/develop (keep jenkins honest on merge conflicts)

**Comment by cuahsimarko** [comment]
@Maurier I think this can be closed

**Comment by cuahsimarko** [comment]
Wrong window -- ignore

**Pull request merged**